api/openapi_server/controllers/auth_controller.py

```diff
@@ -167,13 +168,7 @@ def signUpCoordinator():  # noqa: E501
         msg = f"The parameters you provided are incorrect: {error}"
         raise AuthError({"message": msg}, 500)
 
-        
-
-def signin():
-    # Validate request data
-    if connexion.request.is_json:
-        body = connexion.request.get_json()
-
+def signin(body: dict):
```

Comment:
Should controllers be snake_case? This might be out of scope for this PR


[reference](https://github.com/hackforla/HomeUniteUs/issues/599)

Reply:
Good point. Since I touched it I'll fix the name. [6944201](https://github.com/hackforla/HomeUniteUs/pull/637/commits/69442013820dc721a022ab7c599456d9cddc8eb4)